app/src/main/java/org/simple/clinic/teleconsultlog/shareprescription/TeleconsultSharePrescriptionRepository.kt

```diff
@@ -0,0 +1,58 @@
+package org.simple.clinic.teleconsultlog.shareprescription
+
+import android.app.Application
+import android.content.ContentValues
+import android.graphics.Bitmap
+import android.net.Uri
+import android.os.Build
+import android.os.Environment
+import android.provider.MediaStore
+import androidx.annotation.RequiresApi
+import org.simple.clinic.util.UserClock
+import java.io.File
+import java.io.FileOutputStream
+import java.io.OutputStream
+import java.time.LocalDate
+import javax.inject.Inject
+
+
+class TeleconsultSharePrescriptionRepository @Inject constructor(
+    private val appContext: Application,
+    private val userClock: UserClock
+) {
+
+  fun saveViewAsImage(bitmap: Bitmap?) {
+    val fos: OutputStream
+    val currentDate = LocalDate.now(userClock)
+    val fileName = "simple_prescription_$currentDate.png"
```

Comment:
Updated the signature file name to `Simple prescription 7 Oct 2020 7-18-PM.png` ie without colons. API 21 sends an error that says `EINVAL (Invalid argument)` because we cannot include some special characters into the file name.

Reply:
Can we still change the name to make it more human friendly? We should be adding the time component as well.

"Simple prescription 15th Oct 2020 11.15.30 AM.png"